packages/container/libraries/strongly-typed/src/container.spec.ts

```diff
@@ -155,45 +154,21 @@ describe('interfaces', () => {
         });
 
         it('defaults a child to have an `any` map', () => {
-          const child = container.createChild();
+          const child = new TypedContainer({ parent: container });
           child.bind('unknown').toConstantValue('unknown');
           expect(child.get('unknown')).toBe('unknown');
         });
 
-        it('automatically extends parent types', () => {
-          const child = container.createChild<{ childProp: string }>();
+        it('allows a TypedContainer parent', () => {
+          const child = new TypedContainer<{ childProp: string } & BindingMap>({
```

Comment:
We can possibly make these types stronger/smarter again in future, but I'm not sure I have the time to work it out right now. For now, it's left to consumers to correctly combine the child and parent binding maps.